smart-embed/v1.0/view-key-data.md

```diff
@@ -0,0 +1,74 @@
+# Data required for creating a view key
+
+_Please feel free to create an issue if you have any questions not answered by this documentation._
+
+## Overview
+This page outlines what data is expected when creating a view key as part of the [Video Playback](video-playback.md) workflow.
+
+The `data` property on the create view key request will be used to pass user information to ClickView. Below is a generic outline of the expected properties, however the data being sent here will be an agreement between you and ClickView, additional properties outside of what is specified may be expected by ClickView. Similarly certain properties may be required that are listed as optional below.
+
+### Schema
+
+|Property Name|Required for teachers|Required for students|Description|
+|---|---|---|---|
+|`student`|Required|Required|A boolean value identifying if the user is a student.|
+|`firstName`|Required|Must not be sent|The users first name.|
+|`surname`|Required|Must not be sent|The users surname.|
+|`email`|Required|Must not be sent|The user's email address.|
+|`jobTitle`|Optional|n/a|The user's job title in their school or district.|
+|`schoolName`|Optional|Optional|The name of the school the user belongs to.|
+|`schoolId`|Optional|Optional|Any identifier you may have on hand for the school.|
+|`districtName`|Optional|Optional|The name of the district the user belongs to.|
+|`districtId`|Optional|Optional|Any identifier you may have on hand for the district.|
```

Comment:
Wondering if we should also ask for `userId`. Similar to how we're asking for school and district ids. Even if they don't have anything we understand, we can tie multiple requests to the same id easily, without relying on their email never changing.

We could then theoretically also take `userId` on student?